CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.8.0] - 2020-10-29
+- Empty commit to bump pegasus minor version
```

Comment:
Please add more contexts.

Reply:
I added a little bit more context, but still want to keep it short in the commit history:
1. Commit history is meant to be short, if they want any more context, they should come to this PR description.
2. Pegasus is open-source, I don't think it's proper to mention all the HI context in this open source.